### PR TITLE
Update description of the texts shadow property

### PR DESCRIPTION
**Pull request by DelazJ**
to mimic the previous sections structure

### CODE REVIEW

File: source/docs/user_manual/style_library/label_settings.rst [hunk 1]
```diff
@@ -234,6 +250,14 @@ can also be altered by choosing a blend mode.
    top of the shadow (depending upon the shadow's color), when that setting is
    used.
 
+* Define the shadow's :guilabel:`Opacity`
+* Rescale the shadow's size based on the label's using the :guilabel:`Scale`
```
Comment (havatv) [inline source/docs/user_manual/style_library/label_settings.rst]:
```suggestion
* Rescale the shadow's size using the :guilabel:`Scale`
```

Comment (DelazJ) [reply source/docs/user_manual/style_library/label_settings.rst]: scale factor is a ratio. What is used as base reference does not look obvious to me, then.

Comment (havatv) [reply source/docs/user_manual/style_library/label_settings.rst]: I thought that was obvious.  But if it is not, we could try:
> * Rescale the shadow's size by multiplying with the :guilabel:`Scale`

**Comment by havatv** [comment]
Is this not ready to be merged, @DelazJ?

**Pull request merged**